universal-login-relayer/src/integration/sql/services/EmailConfirmationsStore.ts

```diff
@@ -0,0 +1,25 @@
+import Knex from 'knex';
+import {EmailConfirmation} from '@unilogin/commons';
+
+export class EmailConfirmationsStore {
+  private tableName = 'email_confirmations';
+
+  constructor(private database: Knex) {
+  }
+
+  add(emailConfirmation: EmailConfirmation) {
+    return this.database
+      .insert(emailConfirmation)
+      .into(this.tableName)
+      .returning('email');
+  }
+
+  async get(email: string): Promise<EmailConfirmation> {
```

Comment:
We will need created_at record too. Because we will have to check if a request is not outdated

Reply:
Done